documentation/configure.markdown

```diff
@@ -200,17 +203,11 @@ PuppetDB can use either **a built-in HSQLDB database** or **a PostgreSQL databas
 >
 > Depending on demand, Oracle support may be forthcoming in a future version of PuppetDB. This hasn't been decided yet.
 
-### Using Built-in HSQLDB
-
-To use an HSQLDB database at the default `<vardir>/db`, you can simply remove all database settings. To configure the DB for a different location, put the following in the `[database]` section:
+### HSQLDB Deprecation {#hsqldb-deprecation}
```

Comment:
I feel like giving this its own section is more attention than it deserves. Would it be better to just stick this blurb in a **Note** under the previous header?